### PR TITLE
Adding Initial Overview

### PR DESCRIPTION
**Pull request by magowan**
Adding Initial Overview covering Why/How, What.

Signed-off-by: James Magowan magowan@uk.ibm.com

### CODE REVIEW

File: Overview.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+
+# Confidential Containers
+We are interested in integrating existing [Trusted Execution Environments](https://en.wikipedia.org/wiki/Trusted_execution_environment) (TEE) 
+infrastructure support and technologies with the cloud native world. Our focus is to place a kubernetes pod into a TEE.
```
Comment (fitzthum) [inline Overview.md:4]: Are Confidential Containers inherently connected to pod-centric virtualization? Could we support other runtimes that isolate one container at a time?

Comment (fitzthum) [inline Overview.md:4]: In the first sentence do we want to mention 1) Transparent deployment of unmodified containers 2) Support for multiple hardware platforms 3) A strict trust model that separates the CSP from guest applications

**Review by fitzthum — COMMENTED** [review]
looks pretty good. I wonder if we can be even clearer about the highlights and what the best way to introduce pods is.

File: Overview.md (hunk 1 above)
Comment (magowan) [reply Overview.md]: Thanks, updated this section to try annd include more detail around this

**Review by fitzthum — APPROVED** [review]
LGTM. I agree that maybe we can extend the section about the virtualization boundary with some diagrams at some point, maybe in another doc.

**Pull request merged**